core/common/src/main/java/alluxio/security/authorization/Permission.java

```diff
@@ -46,9 +46,9 @@
    * @param mode the {@link Mode}
    */
   public Permission(String owner, String group, Mode mode) {
-    Preconditions.checkNotNull(owner, ExceptionMessage.OWNER_IS_NULL.getMessage());
-    Preconditions.checkNotNull(group, ExceptionMessage.GROUP_IS_NULL.getMessage());
-    Preconditions.checkNotNull(mode, ExceptionMessage.MODE_IS_NULL.getMessage());
+    Preconditions.checkNotNull(owner, "Owner cannot be null when constructing Permission");
```

Comment:
this is not great ... the whole point of having `ExceptionMessage` class is to aggregate all exception messages there